src/components/date-input/DateInput.tsx

```diff
@@ -18,21 +21,23 @@ interface IDateInputContext {
 
 const DateInputContext = createContext<IDateInputContext>({
   isDateInput: false,
-  registerRef: () => {},
+  registerRef: () => undefined,
   name: '',
   autoCompletePrefix: '',
 });
 
 type DateInputType = 'Day' | 'Month' | 'Year';
 interface DateInputInputProps extends HTMLProps<HTMLInputElement> {
   dateInputType: DateInputType;
+  error?: string;
```

Comment:
In this case, does the error type want to be a string or boolean? If the string isn't used anywhere, it would make a lot more sense as a boolean value

Reply:
Agreed, changed to a boolean and added